### ISSUE TITLE
parse `port` and `backend.port` as numbers

### ISSUE THREAD

**Issue by gr2m** [eHealthAfrica/kazana-config#7]
it currently returns strings from `.kazanarc` files

**Comment by jo** [comment]
In rc they suggest to handle it on the application level:
> Since ini, and env variables do not have a standard for types, your application needs be prepared for strings.

I think we can close this, right?

**Comment by gr2m** [comment]
I'd say we can do this on kazana-level, as `port` and `backend.port` are default kazana options, but handle app-specific config parsing on app level as recommended by rc

**Comment by jo** [comment]
Yes, I meant this is already handled in [kazana](https://github.com/eHealthAfrica/kazana/blob/master/bin/server#L41) (in my case the app layer) and does not need to be handled here in `kazana-config`.

I remember we had a discussion about parsing the ini like command line arguments, thats what I was referring.

**Comment by gr2m** [comment]
> Yes, I meant this is already handled in kazana (in my case the app layer) and does not need to be handled here in kazana-config

But we load `kazana-config` also at other places, and risk is that we parse the numbers in one place, but forget to do it in others. 

> I remember we had a discussion about parsing the ini like command line arguments, thats what I was referring.

Yeah, I'd like to look into it.

**Comment by gr2m** [comment]
As we have namespaces in kazana-config now, this is no longer blocked
